ansible/zapd_cron.yml

```diff
@@ -4,6 +4,14 @@
   become: yes
   become_user: root
 
+  vars:
+   dest_email: alerts@zap.me
+   min_zap: 5000
+   max_zap: 10000
+   testnet_node: testnet1.wavesnodes.com
+   mainnet_node: nodes.wavesnodes.com
+
```

Comment:
waves_node: "{{ 'testnet1.wavesnodes.com' if TESTNET else 'nodes.wavesnodes.com' }}"

we have a variable named TESTNET which is set in ansible_deploy.sh

Reply:
actually call this variable "remote_waves_node" or something like that

Reply:
not sure why that variable is still showing in the zapd_cron.yml. I deleted this before the push. This is being pushed from the ansible_deploy.sh script.